tests/nimble_l2cap_server/nimble.inc.mk

```diff
@@ -27,10 +27,12 @@ CFLAGS += -DAPP_CID=$(APP_CID)
 USEPKG += nimble
 MSYS_CNT ?= 40
 CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MAX_NUM=1
-CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MPS=250
+CFLAGS += -DMYNEWT_VAL_BLE_L2CAP_COC_MPS=251
```

Comment:
Are there not already defined?

Reply:
no. This example is not including the `nimble_netif` module, so all changes in the file above to not apply for this test application...

Reply:
Do the same arguments apply for the chosen values? If you can think of a central place to put the reasoning please do, but otherwise please add the same justification here as well then.

Reply:
no, can't really think of a central place right now. But will add the justification.